torch_xla/csrc/device.h

```diff
@@ -15,7 +15,7 @@ namespace torch_xla {
 // TODO(yeounoh) `SPMD` is a virtual device that defers data `TransferToDevice`
 // until after the paritioning pass. This avoids transfering  the full input
 // tensor to the device.
-enum class XlaDeviceType { CPU, CUDA, TPU, NEURON, SPMD, PLUGIN };
+enum class XlaDeviceType { CPU, CUDA, TPU, NEURON, SPMD, PLUGIN, AOT };
```

Comment:
Do we have a parallel solution on the inductor side?

Reply:
Unfortunately no, AOT cross-device can't be applied to GPU because GPU compile needs to be done on a GPU machines to get auto-tuning results.